newdle/client/src/components/CreateNewdle.js

```diff
@@ -66,8 +60,9 @@ function ParticipantsPage() {
 function TimeSlotsPage() {
   const dispatch = useDispatch();
   const participants = useSelector(getMeetingParticipants);
+  const timeslots = useSelector(getFullTimeslots);
   return (
-    <div className={styles['time-slots-grid']}>
+    <div>
```

Comment:
Do we need that div in the end?

Reply:
Yes, I mentioned it once in the channel. Adding the `div` adds some extra padding to the entire component that otherwise is not there because of this css:
```
main {
  > * {
    padding: 3em;
  }
}
```
inside `App.module.scss`

Reply:
Ok, good to know ;)